generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -164,8 +159,7 @@ Scenario: Running a 'containingRegex' request that includes special characters (
        | "😘" |
 
 Scenario: Running a 'containingRegex' request that includes anchors ^ and $ should be successful
-     Given there is a field foo
-       And foo is containing regex /^[a-c]{1}$/
+     Given foo is containing regex /^[a-c]{1}$/
```

Comment:
shouldn't this regex be `/^$/` ?

Reply:
@mrspaceman would the description for the scenario be more meaningful to be written as:
containingRegex that includes the opening and closing anchors should be successful?

Reply:
I misread the description as `only includes anchors ^ and $`
doesn't need changing